File: openresty/cdn-system/api/models/acl.go
package models

import "time"

// ACL 对应数据库中的 `acl` 表
type ACL struct {
	ID            int64     `json:"id" gorm:"primaryKey"`
	UserID        int64     `json:"uid" gorm:"column:uid"`
	Name          string    `json:"name"`
	Description   string    `json:"des" gorm:"column:des"`
	DefaultAction string    `json:"default_action"`
	Data          string    `json:"data"` // MEDIUMTEXT JSON
	Enable        bool      `json:"enable"`
	TaskID        int64     `json:"task_id"`
	Version       int       `json:"version"`
	CreatedAt     time.Time `json:"create_at" gorm:"column:create_at"`
	UpdatedAt     time.Time `json:"update_at" gorm:"column:update_at"`
}

func (ACL) TableName() string {
	return "acl"
}

File: cnn.net/src/Cnn.Common/Domain/Entities/Acl.cs
using SqlSugar;

namespace Cnn.Domain.Entities;

[SugarTable("acl")]
public class Acl
{
    [SugarColumn(IsPrimaryKey = true, IsIdentity = true)]
    public int Id { get; set; }

    public int? Uid { get; set; }

    public string? Name { get; set; }

    public string? Des { get; set; }

    public string? DefaultAction { get; set; }

    public string? Data { get; set; }

    public DateTime? CreateAt { get; set; }

    public DateTime? UpdateAt { get; set; }

    public bool? Enable { get; set; }

    public long? TaskId { get; set; }

    public int? Version { get; set; }
}